File: Programmers/Lv.0/문자 개수 세기/181902.java
class Solution {
    public int[] solution(String my_string) {
        int[] answer = new int[52];
        
        for (int i=0; i<my_string.length(); i++) {
            int n = (int) my_string.charAt(i) - 65;
            answer[(n > 25) ? (n - 6) : n]++;
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/문자 개수 세기/181902.js
function solution(my_string) {
    const answer = new Array(52).fill(0);
    
    for (let i=0; i<my_string.length; i++) {
        const n = my_string.charCodeAt(i) - 65;
        answer[(n > 25) ? (n - 6) : n]++;
    }
    
    return answer;
}
